config/samples/operator_v1alpha1_externaldns.yaml

```diff
@@ -1,5 +1,15 @@
 apiVersion: externaldns.olm.openshift.io/v1alpha1
 kind: ExternalDNS
 metadata:
-  name: externaldns-sample
+  name: sample
+  namespace: external-dns-operator
```

Comment:
I'll wait for @Miciah to weigh in, but I think you should use `openshift-externaldns-operator` for the namespace

Reply:
I was thinking about it too. However OpenShift gave me an error while I tried to create `openshift-external-operator`  namespace for external-dns-opertor and I was cluster-admin. It seems to be that there is some hard coded check for `openshift-*` namespaces.

Reply:
Yeah, the operator should use the `openshift-externaldns-operator` namespace.  What was the error?

Reply:
My bad, I naively tested the namespace creation with `oc new-project` which gave me this error:
```bash
$ oc new-project openshift-externaldns-operator
Error from server (Forbidden): project.project.openshift.io "openshift-externaldns-operator" is forbidden: cannot request a project starting with "openshift-"
```
Will change the namespace to `openshift-externaldns-operator`.